src/www/js/form.js

```diff
@@ -472,6 +472,7 @@ function displayButtons(appPath, url) {
       window.open(url);
     }
   });
+  btnLive.click();
```

Comment:
Why?

Reply:
When we'll click generate webapp, as suggested by mario in the meeting, the preview must also open in a separate new tab automatically. Let me know if we do not want. @iamareebjamal

Reply:
Is this mentioned in the issue?

Reply:
no, opening the preview is not mentioned in the issue.

Reply:
Then please remove it